acceptance-tests/src/test/java/tech/pegasys/ethsigner/tests/multikeysigner/AzureBasedTomlLoadingAcceptanceTest.java

```diff
@@ -46,7 +46,7 @@ void azureSignersAreCreatedAndExpectedAddressIsReported(@TempDir Path tomlDirect
   }
 
   @Test
-  void incorrectlyNamedAzureFileIsNotLoaded(@TempDir Path tomlDirectory) {
+  void arbitrarilyNamedAzureFileIsLoaded(@TempDir Path tomlDirectory) {
```

Comment:
Think this now misnamed? Don't think the azure file should be loaded

Reply:
that is probably true ...